Formula/cpufetch.rb

```diff
@@ -0,0 +1,18 @@
+class Cpufetch < Formula
+  desc "Simple yet fancy CPU architecture fetching tool"
+  homepage "https://github.com/Dr-Noob/cpufetch"
+  url "https://github.com/Dr-Noob/cpufetch/archive/v1.00.tar.gz"
+  sha256 "2254c2578435cc35c4d325b25fdff4c4b681de92cbce9a7a36e58ad58a3d9173"
+  license "MIT"
+  head "https://github.com/Dr-Noob/cpufetch.git", branch: "master"
+
+  def install
+    system "make"
+    bin.install "cpufetch"
+    man1.install "cpufetch.1"
+  end
+
+  test do
+    system "cpufetch", "-d"
```

Comment:
Can we check the output here?

Reply:
The output is system-dependent, so it can't be checked easily; at the very least, it tests the installed binary. @nandahkrishna mentioned that `neofetch` similarly tests its install this way.

Reply:
There is some output that doesn't change no matter what the system configuration is so we might be able to work with that, as I mentioned in https://github.com/Homebrew/homebrew-core/pull/84631#discussion_r702225538. Perhaps we can ensure that a sane number of cores is output, for example.